x-pack/plugins/cases/server/connectors/cases/constants.ts

```diff
@@ -12,8 +12,11 @@ export const MAX_OPEN_CASES = 10;
 export const DEFAULT_MAX_OPEN_CASES = 5;
 export const INITIAL_ORACLE_RECORD_COUNTER = 1;
 
-export const VALUES_FOR_CUSTOM_FIELDS_MISSING_DEFAULTS: Record<CustomFieldTypes, string | boolean> =
-  {
-    [CustomFieldTypes.TEXT]: 'N/A',
-    [CustomFieldTypes.TOGGLE]: false,
-  };
+export const VALUES_FOR_CUSTOM_FIELDS_MISSING_DEFAULTS: Record<
+  CustomFieldTypes,
+  string | boolean | number
+> = {
+  [CustomFieldTypes.TEXT]: 'N/A',
+  [CustomFieldTypes.TOGGLE]: false,
+  [CustomFieldTypes.NUMBER]: 0, // CHECK!!!
```

Comment:
This is for when a case exists before a custom field is created right?

I'm ok with `0` 👍